client/src/Client.cpp

```diff
@@ -17,13 +16,13 @@ int main()
 
     sge::sgeInit();
     // comment out ModelComposite stuff if you're debugging networking
-    // todo: We prolly need a separate function to init game entities -- there'll be a lot --- combine with ECS @Matthew
-    for (int i = 0; i < NUM_MOVEMENT_ENTITIES; i++) {
-        // ModelComposite
-        playersModel[i] = new sge::ModelComposite("rock_w_tex/rock2.obj");
+    sge::loadModels();
+
+    // TODO: change this when yall want to
+    entities.push_back(std::make_unique<sge::EntityState>(MAP));
+    for (unsigned int i = 0; i < 4; i++) {
+        entities.push_back(std::make_unique<sge::DynamicEntityState>(PLAYER_0, i));
```

Comment:
nit: maybe add a comment saying `i = index into every game data array in ClientGame`